Modules/Example3/cmdapps/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+option(BUILD_CaPTkExample3CmdApp "Build command-line app for CaPTkExample3" ON)
+
+if(BUILD_CaPTkExample3CmdApp)
+  mitkFunctionCreateCommandLineApp(
+    NAME CaPTkExample3CmdApp
+    CPP_FILES CaPTkExample3CmdApp.cpp
+    PACKAGE_DEPENDS ITK Qt5|Core
```

Comment:
I don't think there is any qt used in the cmd app.

Reply:
yes, you are right I will create example4 that will have cmd app.